users/README.md

```diff
@@ -2,26 +2,84 @@
 
 ```
 {
```

Comment:
```suggestion
```json
{
```
Please use JSON code block

Reply:
Please resolve the red highlight issue by updating it to valid JSON.

Reply:
@shubham-y  Please check, I have already committed the suggested change

Reply:
What I meant is to change it to like below example
```json
{
  "id": "String",
  "username": "String",
  "first_name": "String",
  "last_name": "String",
  "email": "String",
}
```
So that it is a valid JSON
For reference check this https://github.com/Real-Dev-Squad/website-data-models/tree/main/users-status

Reply:
Oh, got it. Fixed.

Reply:
NIT: I don't think that extra space formatting is needed